### ISSUE TITLE
Question about step-length and getWaitingTime.

### ISSUE THREAD

**Issue by zhaofromying** [eclipse/sumo#10580]
I`m running my sumo simulation for 3600 steps. If I use default step-length, the experiment spent 360s arround. If I set the step-length to 0.1, the experiment spent 85s arround. So the lower step-length means run simulation faster?

At the same time, I want to collect vehicles waiting time by vehicle.getWaitingTime(vid) to evaluate my algorithm. But in default step-length, I got avg_wait_time 1053.1 and 14.07 in step-length = 0.1  . Its a big different between these two results. I saw the explain about getWaitingTime() which times step-length at last in calculation. But the gap is still big after I devide the step-length.

In conclusion, I want to get vehicle`s waiting time in the case of faster simulation. So, how can I get the correct waiting time by setting the step-length to 0.1

SUMO-version: 1.12.0

**Comment by namdre** [comment]
with --step-length 0.1 you will only simulate 360s when calling `traci.simulationStep` 3600 times.
Possibly, there are much fewer vehicles in that time range which speeds up your simulation and also affects average waiting time.
(The more cars you have queued, the bigger is the impact of a jam).

> So, how can I get the correct waiting time by setting the step-length to 0.1

Perform 36000 simulation steps.
